packages/amplify_core/example/README.md

```diff
@@ -0,0 +1,16 @@
+# amplify_core_example
+
+Demonstrates how to use the amplify_core plugin.
+
+## Getting Started
+
+This project is a starting point for a Flutter application.
```

Comment:
We can make this match the other examples, or remove ## Getting Started entirely.

Reply:
Removed the getting started section.